tools/rna_tools/compalignp/tool_dependencies.xml

```diff
@@ -0,0 +1,21 @@
+<tool_dependency>
+    <package name="squid" version="1.9g">
+        <repository name="package_squid_1_9g" owner="iuc" prior_installation_required="True" />
+    </package>
+    <package name="compalignp" version="1.0">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url">http://www.biophys.uni-duesseldorf.de/bralibase/compalignp.tgz</action>
+                <action type="set_environment_for_install">
+                    <repository name="package_squid_1_9g" owner="iuc">
+                        <package name="squid" version="1.9g"/>
+                    </repository>
+                </action>
+                <action type="shell_command">make</action>
+                <action type="set_environment">
+                    <environment_variable name="PATH" action="prepend_to">$INSTALL_DIR</environment_variable>
```

Comment:
Can you please add the following:
` <environment_variable name="COMPALIGNP_ROOT_PATH" action="set_to">$INSTALL_DIR</environment_variable>`